.github/config.yml

```
@@ -0,0 +1,6 @@
+newPRWelcomeComment: |
+  Thank you for making your first PR on `website`! 💖
```

```suggestion
  Thank you for making your first PR on `website`!
```

Or, if you insist on the heart ("U+2764 HEAVY BLACK HEART in explicit plaintext representation" <3 :D):

```suggestion
  Thank you for making your first PR on `website`! ❤︎
```